### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by nhatthm**
I just added this extension to `homebrew/homebrew-php` to make it more convenient for mac users (https://github.com/Homebrew/homebrew-php/pull/2055). And would like to update the README.md to show that to users :)

### CODE REVIEW

**Comment by EVODelavega** [comment]
This extension is no where near stable, though... I'm not entirely sure if it's the right moment to make this extension readily available just yet. I'm currently refactoring the entire code-base, and in doing so, I'm breaking compatibility with the current master branch in a couple of crucial places.

Help with the refactoring process (https://github.com/EVODelavega/phpkafka/tree/feature/topic-objects/kafka_clean is what I have now) is something I would really appreciate a lot ATM, but once I'm finished refactoring the extension, I'll give you a heads-up when I'm merging it into master, then I'll add the homebrew section to the README

Thanks

File: README.md [hunk 1]
```diff
@@ -57,6 +57,17 @@ Download and install [librdkafka](https://github.com/edenhill/librdkafka/). Run
...
 Installing PHP extension:
 ----------
+
+#### With Homebrew (For Mac users)
+
+```bash
+$ brew install php56-kafka
+```
+
+Note: Install `php53-kafka`, `php54-kafka`, `php55-kafka`, `php56-kafka` or `php70-kafka` depends on PHP version on your machine.
```
Comment (EVODelavega) [inline README.md:67]: `php70-kafka` doesn't exist (yet): The current extension is incompatible with PHP7

**Comment by nhatthm** [comment]
Thanks for your feedback, do we have ETA for refactoring? Thanks

**Comment by EVODelavega** [comment]
@casterx I was rather hoping to finish the refactoring _before_ the first stable release of PHP7 (november 2015). Preferably a bit sooner (mid october), to start work on a PHP7 support branch. All help is welcome, and the refactoring shouldn't be too hard to do, seeing as the bulk of the code has already been written.

What I want to have is a basic extension that just works. When I've reached the alpha state, I'll start work on features like ini settings, and hammer out some of the non-supported features (like client managing the consumer offsets). At the same time, I want to start work on PHP7 support (branching of from the alpha branch), and add new features to the PHP7 extension in parallel... again: any help is welcome

**Comment by nhatthm** [comment]
Hi @EVODelavega, could you please share the current status and ETA of possible (for both PHP 5.4 and PHP 7?

And I'd like to report an issue but I couldn't find the Issues tab, could you please tell me how I can reach you for support? PHP was crashed on my local http://pastebin.com/JVRnNYRQ with Kafka ext installed. It was fine when I removed kafka ext

**Pull request closed without merging**